app/grants/views.py

```diff
@@ -836,8 +845,12 @@ def grant_new(request):
         'conf_time_spread': conf_time_spread(),
         'gas_advisories': gas_advisories(),
         'trusted_relayer': settings.GRANTS_OWNER_ACCOUNT,
-        'grant_types': GrantType.objects.all()
+        'grant_types': GrantType.objects.all(),
+        'data': {}
     }
+    if check_profile is not None:
+        params["data"] = project_data
```

Comment:
This seems un-necessary ! Add this logic to L849

Reply:
done

Reply:
Done